salt/modules/pcs.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+'''
+Configure a Pacemaker/Corosync cluster with PCS
+===============================================
+
+Configure Pacemaker/Cororsync clusters with the
+Pacemaker/Cororsync conifguration system (PCS)
+
+:depends: pcs
+
+.. versionadded:: 2016.3.0
+'''
+from __future__ import absolute_import
+
+# Import salt libs
+import salt.utils
+
+
+def __virtual__():
+    '''
+    Only load if pcs package is installed
+    '''
+    if salt.utils.which('pcs'):
+        return 'pcs'
+    return False
+
+
+def auth(nodes, pcsuser='hacluster', pcspasswd='hacluster', extra_args=None):
+    '''
+    Authorize nodes to the cluster
+
+    nodes
+        a list of nodes which should be authorized to the cluster
+    pcsuser
+        user for communitcation with PCS (default: hacluster)
+    pcspasswd
+        password for pcsuser (default: hacluster)
+    extra_args
+        list of extra option for the \'pcs cluster auth\' command
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' pcs.auth nodes='[ node1.example.org node2.example.org ]' \\
+                          pcsuser='hacluster' \\
+                          pcspasswd='hacluster' \\
+                          extra_args=[ '--force' ]
+    '''
+    cmd = ['pcs', 'cluster', 'auth']
+
+    if pcsuser:
+        cmd += ['-u', pcsuser]
+
+    if pcspasswd:
+        cmd += ['-p', pcspasswd]
+
+    if isinstance(extra_args, (list, tuple)):
+        cmd += extra_args
+    cmd += nodes
+
+    return __salt__['cmd.run_all'](cmd, output_loglevel='trace', python_shell=False)
```

Comment:
@cachedout @rallytime: 
Are there rules for return values of module functions?
Would it be better to make this here an internal function (_auth) for use with the state module pcs.auth and create a new function with return values True or False (or whatever are the rules for return values of module functions)?